docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
+
+Difference measures
+^^^^^^^^^^^^^^^^^^^
+
+Difference measures are used to estimate the *excess* risk of disease caused by a risk factor *among the exposed group*. That is, difference measures of effect estimate how much of the
+disease in the exposed group was due to the risk factor of interest. Two commonly used difference measures of effect are the risk difference and the risk difference percent.
+
+*Risk difference* (RD) is the absolute differene between two risks. This is calculated by subracting the risk in the unexposed group (risk0) from the risk in the exposed group (risk1):
+
+Risk difference = risk in exposed (risk1) - risk in unexposed (risk0)
+
+Similary, the rate difference is calculated by subtracting the rate in the unexposed from the rate in the exposed. 
+
+For example, A study measured the risk of HIV infection among children born to HIV-infected mothers,according to whether the babies were breastfed or not. Among non-breastfed children of HIVinfected
+mothers, the risk of HIV infection was 150 infections per 1000 children. Among breastfed babies, the risk was 280 infections per 1000 children. The risk difference was thus
+130 infections per 1000 children (130 = 280 - 150). The interpretation is that the risk factor, in this case breastfeeding, was responsible for the infection of 130 of every 1000 children
+born to, and breastfed by, HIV-infected mothers. Notice that the risk difference retains the same units as the original risks used to calculate it. Thus, if the risk in the exposed and
+unexposed groups is measured in ‘cases per 1000 persons’, then the risk difference will have the same units.
+
+In most situations, where disease is not very common, risk differences and rate differences will be numerically similar. (Note that in the above example, HIV infection was common among study participants, so risk and rate differences would be unlikely to be similar.) In the literature, the risk difference is sometimes called the *attributable risk* or *excess risk*. Similarly, the terms attributable rate or excess rate are sometimes used to mean the rate difference.
+
+The *risk difference percent* (RD%) measures the proportion of cases in the exposed group that are due to the exposure. That is, the RD% is the excess risk among the exposed expressed as a
+proportion (or percentage) of the risk in the exposed group. It is calculated by dividing the risk difference by the risk among the exposed: 
+
+Risk difference % = (risk1-risk0)/risk1
+
+For example, the RD% from the above example is (280/1000 -150/1000)/ 280/1000 = 0.46 or 46%
+
+We interpret this by saying breastfeeding was responsible for 46% of HIV infections among children born to, and breastfed by, HIV-infected mothers (the exposed). Note that this
```

Comment:
@Ninicorn you had a really nice summary sentence last week about when to use difference vs ratio.

We were looking at smoking risk for CHD by age, and finding that while there was a stronger connection between the two in younger age groups, the rates were so high in older age groups that one could avert far more CHD cases by eliminating the smoking in older ages.

**So you mentioned that rate difference is frequently of more interested in policy/decision-making contexts, and rate ratios are of more interest in....** i forget. a scientific bio-mechanical context?

I think that would fit really nicely here!